src/components/tile/tile.stories.mdx

```diff
@@ -202,38 +202,46 @@ Please note that it will not have any effect if the Tile orientation is set to `
 
 <Canvas>
   <Story name="custom Widths">
+    <div>
     <Tile variant="tile" orientation="horizontal" width="75%">
+        <Content title="Test Title">
```

Comment:
It looks like you forgot to move these stories to `tile.stories.tsx` file

Reply:
Nevermind, after discussion with Kuba, he showed me that it is impossible to refactor these stories with the current implementation of this component - Kuba will open a separate ticket to fix the buggy implementation